packages/babel-register/test/cache.js

```diff
@@ -40,9 +42,16 @@ describe("@babel/register - caching", () => {
       load = cache.load;
       get = cache.get;
       save = cache.save;
+
+      consoleWarnSpy = jest
+        .spyOn(console, "warn")
+        .mockImplementation(err => (stderr += err));
```

Comment:
Can you add a newline, in case `console.warn` gets called multiple times? `err + "\n"`

Reply:
Oh I realized that `stderr` is not used throughout the test. I am down to removing it.